src/main/java/seedu/address/logic/commands/AddTodoCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.CompletableTodo;
+import seedu.address.model.task.todo.Todo;
+
+public class AddTodoCommand extends Command {
+
+    public static final String COMMAND_WORD = "addTto";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a Task to CoLAB. "
+            + "Parameters: "
+            + PREFIX_DESCRIPTION + "DESCRIPTION "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_DESCRIPTION + "Submit project report ";
+
+    private final Index index;
+    private final Todo toAdd;
+
+    /**
+     * Creates an AddTodoCommand to add the specified {@code Task}
+     * @param index index of {@code Project} to add event in the list.
+     * @param todo {@code Todo} to add.
+     */
+    public AddTodoCommand(Index index, Todo todo) {
+        requireNonNull(todo);
+        this.index = index;
+        this.toAdd = todo;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Project> lastShownList = model.getFilteredProjectList();
+
+        if (index.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PROJECT_DISPLAYED_INDEX);
+        }
+
+        if (index.getZeroBased() < 0) {
```

Comment:
As Samueal commented on my PR before, the getZeroBased method alr checks for negative value, so this is unnecessary

Reply:
Oh nice catch. thanks @Eriksen2411 !